server/bin/pbench-server-put-shim

```diff
@@ -0,0 +1 @@
+pbench-trampoline
```

Comment:
I wish GitHub wouldn't make me click through to the file to confirm that this is a softlink and not a real file with the text `pbench-trampoline`. 😆

Reply:
You should consider reporting that desire to the GitHub folks...you might get your wish, that way.